src/Compilers/VisualBasic/Portable/LanguageVersion.vb

```diff
@@ -13,6 +13,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         VisualBasic12 = 12
         VisualBasic14 = 14
         VisualBasic15 = 15
+        VBvNext = Integer.MaxValue
```

Comment:
Please condition the feature on a /feature flag instead. This is an unwanted public API change.

---
In reply to: [72991961](https://github.com/dotnet/roslyn/pull/12629#discussion_r72991961) [](ancestors = 72991961)